src/controllers/QueryRunner.ts

```diff
@@ -198,6 +173,7 @@ export default class QueryRunner {
 
         // Store the batch again to get the rest of the data
         this._batchSets[batch.id] = batch;
+        this._totalElapsedMilliseconds += new Date(batch.executionEnd).getTime() - new Date(batch.executionStart).getTime();
```

Comment:
An unfortunate oversight. The elapsed time is a string and a JS Date can't be instantiated from a TimeSpan string (eg, 00:01:00.00). So, I'm stuck recalculating it, for now.